remove.xml

```diff
@@ -144,4 +144,8 @@
   <remove-project name="platform/system/vold" />
   <remove-project name="platform/system/sepolicy" />
 
+<!-- Add Team-WIN-Recovery-Project -->
+  <include name="multirom.xml" />
```

Comment:
This dependency should not be added in the remove.xml.
It should be added in omni-default instead, at the end, before remove-minimal.
Also the comment should be more generic, like the other ones, to add MultiROM specific changes.